test/subspace-test-runtime/src/lib.rs

```diff
@@ -136,7 +136,7 @@ pub const MILLISECS_PER_BLOCK: u64 = 6000;
 
 // NOTE: Currently it is not possible to change the slot duration after the chain has started.
 //       Attempting to do so will brick block production.
-const SLOT_DURATION: u64 = 1000;
+const SLOT_DURATION: u64 = 300;
```

Comment:
I would recommend to decrease this to 100ms and above constant with block time to 100ms too. Then below ratio should be `(1, 1)`, not `(1, 6)`.

Reply:
In my local experiment, the block production can stall due to insufficient block proposal time with an overloaded machine if using 100ms, so 300ms is a kind of conservative but _safe_ value, I'm not sure if 100ms is okay.

The ratio actually does not help here as it's only used to calculate the solution range when enacting the era change in consensus, which is dominant in the tests. But having a right parameter is not bad even it does not take affect :P

Reply:
Okay, then let's make block time constant 300ms and still fix the ratio